collect_stata/write_json.py

```diff
@@ -202,10 +202,44 @@ def generate_statistics(
 
     return metadata
 
+def update_metadata(
```

Comment:
This would better fit into some of the other modules.
https://en.wikipedia.org/wiki/Cohesion_(computer_science)